router/middleware.go

```diff
@@ -91,14 +94,19 @@ func (m *Middleware) RequireAuthorization() gin.HandlerFunc {
 	}
 }
 
-// Helper function to fetch a server out of the servers collection stored in memory.
-//
-// This function should not be used in new controllers, prefer ExtractServer where
-// possible.
-func GetServer(uuid string) *server.Server {
-	return server.GetServers().Find(func(s *server.Server) bool {
-		return uuid == s.Id()
-	})
+func (m *Middleware) WithServerManager() gin.HandlerFunc {
+	return func(c *gin.Context) {
+		c.Set("servermanager", m.serverManager)
+	}
+}
+
+func ServerManagerFromContext(c *gin.Context) server.Manager {
```

Comment:
I've been using `Extract___` as the naming for these guys, e.g. `ExtractServerManager()`

Reply:
Yeah I saw, I guess I'll rename it